dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/connector/cloudera/manager/AbstractClouderaTimeSeriesTask.java

```diff
@@ -59,7 +60,15 @@ protected JsonNode requestTimeSeriesChart(ClouderaManagerHandle handle, String q
     CloseableHttpClient httpClient = handle.getHttpClient();
     JsonNode chartInJson;
     try (CloseableHttpResponse chart = httpClient.execute(new HttpGet(uriBuilder.build()))) {
-      chartInJson = objectMapper.readTree(chart.getEntity().getContent());
+      int statusCode = chart.getStatusLine().getStatusCode();
+      if (!isStatusCodeOK(statusCode)) {
+        throw new TimeSeriesException(statusCode, "Expected status code is 2xx.");
```

Comment:
expected 2xx, but got ???
I think we need to store responses somewhere in such cases.

Reply:
Sure! Thanks.